ouroboros-consensus/test-consensus/Test/Consensus/Mempool.hs

```diff
@@ -202,35 +328,51 @@ data TestSetup = TestSetup
   { testLedgerState :: LedgerState TestBlock
   , testInitialTxs  :: [TestTx]
     -- ^ These are all valid and will be the initial contents of the Mempool.
+  , testMempoolCap  :: MempoolCapacity
   } deriving (Show)
 
 ppTestSetup :: TestSetup -> String
-ppTestSetup TestSetup { testLedgerState, testInitialTxs } = unlines $
+ppTestSetup TestSetup { testLedgerState
+                      , testInitialTxs
+                      , testMempoolCap = MempoolCapacity mpCap
+                      } = unlines $
     ["Ledger/chain contains TxIds:"]         <>
     (map condense (tlTxIds testLedgerState)) <>
     ["Initial contents of the Mempool:"]     <>
-    (map condense testInitialTxs)
+    (map condense testInitialTxs)            <>
+    ["Mempool capacity:"]                    <>
+    [condense mpCap]
 
 -- | Generate a 'TestSetup' and return the ledger obtained by applying all of
 -- the initial transactions.
-genTestSetup :: Int -> Gen (TestSetup, LedgerState TestBlock)
-genTestSetup n = do
-    ledgerSize   <- choose (0, n)
-    nbInitialTxs <- choose (0, n)
+genTestSetup :: Int -> Int -> Gen (TestSetup, LedgerState TestBlock)
+genTestSetup maxInitialTxs nbNewTxs = do
```

Comment:
Updated the docstring for `genTestSetup`.

As it relates to `nbNewTxs`, I've been debating this for a bit and also tried to come up with better names. In my opinion, `nbNewTxs` seems clear in this context as it represents the number of new transactions that we'll need room for in the mempool.

Maybe it would be more clear if we instead accept the `MempoolCapacity` as a parameter (as opposed to trying to come up with a good name for this operand involved in the calculation of the `MempoolCapacity`)?

Reply:
Maybe `extraCapacityForNewTxs`?

Reply:
Went with `extraCapacity`.